website/api/controllers/entrance/send-password-recovery-email.js

```diff
@@ -55,7 +65,7 @@ module.exports = {
       subject: 'Password reset instructions',
       template: 'email-reset-password',
       templateData: {
-        firstName: userRecord.firstName,
+        passwordResetLocation: passwordResetLocation,
         token: token
       }
```

Comment:
Does this need to exist, or can we use the same password recovery flow for both and leave this code as-is?  I didn't expect to see password recovery change at all as part of this PR, and it was not included in the last set of changes I reviewed, so I'm surprised to see it now.

Reply:
>Does this need to exist, or can we use the same password recovery flow for both and leave this code as-is?

The password recovery flow that was planned for Fleet Sandbox in #5036 has us redirect users to their logged-in Fleet Sandbox after they successfully changed their password.

>I didn't expect to see password recovery change at all as part of this PR, and it was not included in the last set of changes I reviewed, so I'm surprised to see it now.

This is my fault, when I was drafting this PR with code comments, I was going off our (now changed) authentication plan for Fleet Sandbox, and the password recovery flow would only need a redirect (which I had forgotten to include in the draft plan). 
When I started writing the code for this PR, I updated this flow because the requirements changed to POST a login form with the user's credentials to the Sandbox Instance instead of using a generated key stored on the User model.

Reply:
@eashaw Got it, very helpful, thank you.

Instead of adding more pages and actions which have to be maintained, would it work to use one password recovery flow and have it look to see if sandbox is not expired, and if not, redirect to it, else redirect to the license dispenser portal?

Reply:
>Instead of adding more pages and actions which have to be maintained, would it work to use one password recovery flow and have it look to see if sandbox is not expired, and if not, redirect to it, else redirect to the license dispenser portal?

That would work and it would be an easy change. My only concern with this approach is that Fleet Sandbox users with expired instances would be redirected to the [`/customers/new-license` page](https://www.figma.com/file/Jzo81K6E4jC0mcjD4JsWM8/%F0%9F%9A%A7-fleetdm.com-(scratchpad)?node-id=1021%3A2046) with no explanation why. These users would have to navigate back to the Fleet homepage and then click the "try it now" button to see that their sandbox is expired.

Reply:
Ah, I see.  Could we look to see if they have a credit card saved and only link them to license dispenser in that case, otherwise show them the expired sandbox page?

Reply:
Yes! I added the redirect to [`view-login.js`](https://github.com/fleetdm/fleet/pull/6380/commits/2e8e26b47e5d6ccfe80e96ad2e48f24dabdc2601#diff-4deb19da69982f6ce9c1c5616ae0abae51dffb1e3702c8182306e68c15a49139)